palmetto/mapdwell-base-styles
# Allow changing sizes defined in rem

This PR allows rem unit styles to be scaled by a given factor. This allows the styles to translate to the same pixel sizes when used on mapdwell.com or in Fluxboard, even though they now have different root font sizes. The default $rem-factor is 1, which corresponds to a root font size of 15px. To account for a different font size, set the $rem-factor sass variable before importing `mapdwell-base-styles/_styles`.

Trello card: https://trello.com/c/9jBCSQN0/589-restore-rem-sizes

Manual tests:
1. In this directory, run `watchman -- trigger $PWD cr '**/*.scss' -- sh cp_to_mapdwell.sh`. Start the rails server and navigate to http://localhost:5000/en. The h2 element with text "Seizing the future at the grid's edge" has class mb35. Ensure the computed margin-bottom is 22.5px.
2. In this directory, run `watchman -- trigger $PWD cr '**/*.scss' -- sh cp_to_fluxboard.sh`. In the `fluxboard` repo, remove `font-size: 93.75%;` from `app/styles/_base.scss` to set the root font size to 16px. Set `$rem-factor: 15 / 16;` at the top of `app/styles/app.scss`. Start the Ember server and navigate to http://localhost:4200/en/fluxboard/app. The div with class `fluxboard-logo` also has class `ml35-ns`. Ensure the computed margin-left is 22.5px.
3. In Fluxboard, select an unlocked building on the map. Click the + button. Ensure the + icon does not move when the menu opens.
4. Open the address card for a building. Ensure it looks the same as before: the text field label font size should be 15px.
5. Open a list. For each list item, ensure the address font size is 14px.

## Checklist:

- [x] Update version
